src/database.ts

```diff
@@ -17,12 +17,18 @@ export default class StatsDatabase {
    */
   private customEvents: Collection<any>;
 
+  /**
+   * Timing is used to record application metrics that deal with latency.
+   */
+  private timings: Collection<any>;
```

Comment:
I realllly wanted to use an `interface` here, but I kept getting this error message: 

```
Property 'length' is missing in type '{ eventType: string; durationInMilliseconds: number; metadata: {}; date: string; }'.
```

![bitmoji](https://render.bitstrips.com/v2/cpanel/3aed3b46-fa34-4d13-beb6-b5e2d57b807f-f14a1e1d-64bc-41f2-8006-01b6c63adbd8-v1.png?transparent=1&palette=1&width=246)